### ISSUE TITLE
Using the "space: false" option doesn't eliminate the space next to the clear button.

### ISSUE THREAD

**Issue by davidklebanoff** [wbotelhos/raty#185]
Using the "space: false" option doesn't eliminate the space next to the clear button. It is always prepended or appended. The issue is in _createCancel.

**Comment by wbotelhos** [comment]
This options is about stars, not cancel button.
Maybe we can include the cancel space on it.

What do you think?

**Comment by davidklebanoff** [comment]
Yeah either that, or add an option for specifically for the cancel space. However, I don't see why someone would want spaces for one and not the other. So, my personal preference is one option that affects the spacing for both stars and the cancel button.
